h2/src/main/org/h2/engine/DbSettings.java

```diff
@@ -334,6 +335,13 @@
      */
     public final boolean compressData = get("COMPRESS", false);
 
+    /**
+     * Database setting <code>IGNORE_CATALOGS</code>
+     * (default: false).<br />
+     * ignore Catalog-Prefix at object-Names.
```

Comment:
Please, use normal spelling and punctuation here. Also the description should be more correct, now it looks like the catalogs aren't accepted by default, but actually they are always accepted. This setting only allows usage of any names.

Reply:
Would 
"Generally accept the all catalog prefixes at objectnames even if they are not equal to the current databasename."
be better?

Reply:
Use something like
```
If set, all catalog names in identifiers are silently accepted
without comparing them with the short name of the database.
```

Reply:
If ok, I will copy that.